packages/components/src/color-palette/style.scss

```diff
@@ -15,4 +16,28 @@
 	box-sizing: border-box;
 	color: $white;
 	cursor: pointer;
+
+	&::after {
+		content: "";
+		position: absolute;
+		top: -2px;
+		left: -2px;
+		bottom: -2px;
+		right: -2px;
+		box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.2);
+		// Show a thin circular outline in Windows high contrast mode, otherwise the button is invisible.
+		border: 1px solid transparent;
+	}
+
+	&:focus,
+	&[aria-expanded="true"] {
```

Comment:
I'm thinking it would be more consistent if we **don't** add focus styles in the aria-expanded state, since our focus has already moved to the popover. For example the [`DropdownMenu`](https://wordpress.github.io/gutenberg/?path=/story/components-dropdownmenu--default) is a similar pattern.